src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/ButcherAndroidListener.java

```diff
@@ -88,5 +88,9 @@ private void addExtraDrops(List<ItemStack> drops, EntityType entityType) {
         if (entityType == EntityType.BLAZE) {
             drops.add(new ItemStack(Material.BLAZE_ROD, 1 + random.nextInt(1)));
         }
+
+        if (entityType == EntityType.VINDICATOR) {
+            drops.add(new Itemstack(Material.EMERALD, 1 + random.nextInt(2)));
```

Comment:
Need to capitalize stack
```suggestion
            drops.add(new ItemStack(Material.EMERALD, 1 + random.nextInt(2)));
```